### ISSUE TITLE
Re-classify VOC timeseries

### ISSUE THREAD

**Issue by jessicaguo** [az-digitalag/organization#748]
- [x] Finish checking VOC timeseries
- [x] See if new model form will account for exponentially decaying uptake
- [x] Re-run model

**Comment by jessicaguo** [comment]
Finished checking the exponential and Ricker models of the 304 timeseries and after a few small corrections, I am confident that the algorithm correctly predicts the best model fit for either phase for 302. The exceptions are mz = 70.072 (none-Ricker):
![image.png](https://images.zenhubusercontent.com/5f513e94d41e1d43610542ee/6b383440-c9b1-4ed2-b0cc-7b86cb17c959)

And mz = 101.105 (Ricker-Ricker):
![image.png](https://images.zenhubusercontent.com/5f513e94d41e1d43610542ee/08fa6f0d-1f18-477e-8827-09222be19552)

In both cases, the second pulse should probably be exponential rather than Ricker. However, the Ricker function is selected because of minute improvements in Dsum and R2. Rather than changing the model criteria, which would impact all of the other selections, I wonder whether the Ricker model form can be constrained to be below zero if concave up and above zero if concave down:
![image.png](https://images.zenhubusercontent.com/5f513e94d41e1d43610542ee/381bbc56-be43-4485-9bda-0538f2cba5e3)

Rather than concave up above zero:
![image.png](https://images.zenhubusercontent.com/5f513e94d41e1d43610542ee/3e0fe53c-bbca-4eb9-947d-6a1891c7a221)

Basically, so that the indefinite integral is finite. If I can do this somehow in the Ricker function model code (and still have normal run times and convergence), this would be the final part of this puzzle.

**Comment by jessicaguo** [comment]
Ricker model is now constrained so that a and c parameters have the same direction (positive or negative), by using a scaling factor A that has a gamma distribution. This fixes the minor case of 'upside down' Ricker fits that are more appropriately classified as exponential. 

The categorization is now satisfactory and reproducible without too many exceptions.
